osu.Game/Rulesets/Objects/Legacy/ConvertHitObjectParser.cs

```diff
@@ -41,14 +41,14 @@ public HitObject Parse(string text, double offset)
 
                 if (type.HasFlag(ConvertHitObjectType.Circle))
                 {
-                    result = CreateHit(new Vector2(int.Parse(split[0]), int.Parse(split[1])), combo);
+                    result = CreateHit(new Vector2((int)float.Parse(split[0]), (int)float.Parse(split[1])), combo);
```

Comment:
you *must* specify `CultureInfo.InvariantCulture` else this will fail on roughly half of systems.

Reply:
also rather than repeating this code three times, i'd just define it once above with the other variables `Vector2 pos = ...`